conversant/prompt_chatbot.py

```diff
@@ -433,11 +433,19 @@ def configure_client(self, client_config: Dict = {}) -> None:
                 "temperature": 0.75,
                 "frequency_penalty": 0.0,
                 "presence_penalty": 0.0,
-                "stop_sequences": ["\nUser:", self.bot_name],
+                "stop_sequences": ["\\n", "\n"],
             }
         # Override default config values with the config passed in
         if isinstance(client_config, Dict):
-            client_config["stop_sequences"] = ["\nUser:", "\n{}".format(self.bot_name)]
+            if "model" in client_config:
+                if "command" in client_config["model"]:
+                    if (
+                        "\n{}".format(self.bot_name)
+                        not in client_config["stop_sequences"]
+                    ):
+                        client_config["stop_sequences"].append(
+                            "\n{}".format(self.bot_name)
```

Comment:
```suggestion
                            "\n{}:".format(self.bot_name)
```
To be consistent with the stop sequence you used in the historian persona config